src/agentPHPUnit.php

```diff
@@ -72,6 +73,37 @@ public function __destruct()
         self::$httpService->finishAll($HTTPResult);
     }
 
+    /**
+     * @param $launchDescription
+     * @param $envVariables
+     * @return string
+     */
+    private function getLaunchDescriptionFormatted($launchDescription, $envVariables)
+    {
+        if (empty($launchDescription)) {
+            return $this->getVariablesFormatted($envVariables);
+        }
+
+        return $launchDescription . ", " . $this->getVariablesFormatted($envVariables);
```

Comment:
I would add a check in case the `getVariablesFormatted()` function returns an empty string, otherwise there would be an extra comma at the end of the description.